frankenstein/chunk_store.go

```diff
@@ -344,7 +363,11 @@ func (c *AWSChunkStore) Get(ctx context.Context, from, through model.Time, match
 		}
 	}
 
-	return append(fromMemcache, fromS3...), nil
+	// TODO instead of doing this sort, propagate an index and assign chunks
+	// into the result based on that index.
+	chunks := append(fromMemcache, fromS3...)
+	sort.Sort(wire.ChunksByID(chunks))
```

Comment:
For my understanding, is the only reason we're doing this sort here because of tests?